frontend/src/app/help-me-pick/help-me-pick.component.html

```diff
@@ -1,8 +1,11 @@
 <section class="usa-section">
   <div class="usa-grid">
-    <div class="breadcrumbs">
-      <a routerLink="/">Apply for a permit</a> > <span>Help me find a permit</span>
-    </div>
+    <nav class="breadcrumb">
```

Comment:
still a bit confused why some breadcrumbs seem to be more autogenerated than others.

Reply:
same as above